Modules/Module_Import/Import/xASL_imp_DetermineStructureFromSourcedata.m

```diff
@@ -216,16 +234,16 @@ function xASL_imp_PrintRun(x,thisSubject,thisVisit,thisRun)
     
     % Sort by output
     if length(x.importOverview.(sFieldName).visitIDs)>1
-        for iV=1:numel(x.importOverview.(sFieldName).visitIDs)
+		IDrow = zeros(1, numel(x.importOverview.(sFieldName).visitIDs));
```

Comment:
Same here. Initializing this has no true effect on code speed, it just creates more text. I understand that this is good practice and required for lower-level languages like C but just annoying when you need to check code.

And can potentially generate new bugs, e.g., now you have to check that nowhere in the function we check for the existence of IDrow as a variable. I would vote for not implementing accelerations that will create negligible improvements but make the code more complicated.

Reply:
There's one other reason to do it. If you have such a vector that can have potentially variable length, and the vector is reused many time, then it's always good to clean it beforehand to avoid having residual values from the previous loop. So that's why I always try to clean reused variables to avoid these potential bugs. 

I agree that here it wasn't necessary, but I really do this as a pre-caution to avoid bugs rather than for speeding up which would make no difference here.

Reply:
Ah that's actually a good reason, to initialize a clean variable. I was thinking only about the speed increase.
Thanks, good point! Please resolve once you read this.